portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1042,16 +1066,20 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	    return def.promise();
 	};
     })();
+
     
     return {
 	'known_mutation_settings': {
-	    'ignore_unknown': false,
-	    'recognize_cbioportal_count': false,
-	    'cbioportal_count_thresh': 10,
-	    'recognize_cosmic_count': false,
-	    'cosmic_count_thresh': 10,
-	    'recognize_hotspot': true,
-	    'recognize_oncokb_oncogenic': true,
+		'ignore_unknown': false,
+		'recognize_cbioportal_count': false,
+		'cbioportal_count_thresh': 10,
+		'recognize_cosmic_count': false,
+		'cosmic_count_thresh': 10,
+		'recognize_hotspot': !customDriverAnnotationsByDefault,
+		'recognize_oncokb_oncogenic': !customDriverAnnotationsByDefault,
```

Comment:
Yes, this is for starting the checkboxes. So basically, when custom driver annotations is set to `true` in the `portal.properties`, then `customDriverAnnotationsByDefault` is `true`. Therefore, the checkboxes of `binary_filter`, `multi_filter` and the first class/tier of `multi_filter` are checked, and the checkboxes for `hotspot` and `oncokb` are unchecked. If `customDriverAnnotationsByDefault` is `false`, the checked checkboxes are inverted.